setup.cfg

```diff
@@ -1,13 +1,55 @@
-[bdist_wheel]
-universal = 1
+[metadata]
+name = toolchest
+author = Red Hat
+author_email = lhh@redhat.com
+maintainer = Lon Hohberger
+maintainer_email = lon@metamorphism.com
+description = toolchest is a collection of generic functions for release-depot
+long_description = file: README.rst, HISTORY.rst
+long_description_content_type = text/x-rst
+url = https://github.com/release-depot/toolchest
+project_urls =
+    Bug Tracker = https://github.com/release-depot/toolchest/issues
+license = MIT
+license_files = LICENSE
+classifiers =
+    Development Status :: 4 - Best
```

Comment:
Looks like "4" should be Beta, not Best? https://pypi.org/classifiers/

Reply:
Ha, that is a hilarious typo, I'll fix these and update.

Reply:
I thought it might be a typo for "Best effort" at first and went to check :-)